tests/foreman/cli/test_errata.py

```diff
@@ -714,757 +700,768 @@ def test_host_errata_search_commands(self):
-                        'organization-label': self.org['label'],
-                        'order': sort_text,
-                        'per-page': ERRATUM_MAX_IDS_INFO,
-                    }
-                )
-                # note: the erratum_list, contain a list of restraint info
-                # (id, errata-id, type, title) about each errata
-                self.assertGreater(len(erratum_list), 0)
-                # build a list of erratum id received from Erratum.list
-                erratum_ids = [errata['id'] for errata in erratum_list]
-                # build a list of errata-id field in the same order as received
-                # from Erratum.list
-                errata_ids = [errata['errata-id'] for errata in erratum_list]
-                # build a sorted more detailed erratum info list, that also
-                # contain the sort field
-                sorted_errata_info_list = self._get_sorted_erratum_ids_info(
-                    erratum_ids, sort_by=sort_field, sort_reversed=sort_reversed
-                )
-                # build a list of sort field (issued/updated) values in the
-                # same order as received from the detailed sorted erratum info
-                # list
-                sort_field_values = [errata[sort_field] for errata in sorted_errata_info_list]
-                # ensure that the sort field (issued/updated) values are sorted
-                # as needed
-                self.assertEqual(
-                    sort_field_values, sorted(sort_field_values, reverse=sort_reversed)
-                )
-                # build a list of errata-id field value in the same order as
-                # received from the detailed sorted errata info list
-                sorted_errata_ids = [errata['errata-id'] for errata in sorted_errata_info_list]
-                # ensure that the errata ids received by Erratum.list is sorted
-                # as needed
-                self.assertEqual(errata_ids, sorted_errata_ids)
-
-    @pytest.mark.tier3
-    def test_positive_list_filter_by_product_id(self):
-        """Filter errata by product id
-
-        :id: 7d06950a-c058-48b3-a384-c3565cbd643f
-
-        :Setup: Errata synced on satellite server.
+ORG3_YUM_BIG_ERRATUM_COUNT = FAKE_3_YUM_ERRATUM_COUNT
+ORG3_BIG_ERRATA_ID = FAKE_3_ERRATA_ID
 
-        :Steps: erratum list --product-id=<productid>
 
-        :expectedresults: Errata is filtered by product id.
-        """
-        org_product_erratum_list = Erratum.list(
-            {'product-id': self.org_product['id'], 'per-page': 1000}
-        )
-        org_product_errata_ids = {errata['errata-id'] for errata in org_product_erratum_list}
-        errata_org_product_erratum_list = Erratum.list(
-            {'product-id': self.errata_org_product['id'], 'per-page': 1000}
-        )
-        errata_org_product_errata_ids = {
-            errata['errata-id'] for errata in errata_org_product_erratum_list
+@pytest.fixture(scope='class')
+def org1():
```

Comment:
Again please check your indentation here, there are fixtures defined under a class, but not indented.

Reply:
Hi

I have removed the classes
I have reworked the fixtures, please review again